src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -3,7 +3,7 @@
 import org.hibernate.SessionFactory;
 import org.hibernate.cfg.Configuration;
 
-public class HibernateUtil {
+public final class HibernateUtil {
```

Comment:
Why do you need modify this class?

Reply:
As a matter of best practice, utility classes should be declared as final to prevent inheritance.
I automatically declared it final to prevent it from being subclassed.
I can delete changes, it doesn't matter in this case.